docs/tutorials/how-to-test-a-contract.md

```diff
@@ -105,34 +86,17 @@ describe('Test SmartContract `Demo`', () => {
 
 ## Build and Run tests
 
+Alright, now that we know what our generated tests are made of we will try to run them. 
```

Comment:
made of we -> made of, we

To do that, we just simply run a single command: -> We just run a single command: